libs/ledger-live-common/src/families/casper/bridge/bridgeHelpers/addresses.ts

```diff
@@ -40,7 +40,7 @@ export function casperAccountHashFromPublicKey(
 }
 
 export function casperAddressFromPubKey(pubkey: Buffer, keySig: CLPublicKeyTag): string {
```

Comment:
Can you provide unit test for this ? :pray:

Reply:
Hi @hedi-edelbloute  check this commit for the test https://github.com/LedgerHQ/ledger-live/pull/7728/commits/9635bb83ce809b6b105833e6bb7a055a4fade97e

Reply:
Perfect :+1: